library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -218,6 +219,19 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Specify whether to use the getaddr-based system DNS resolver or the c-ares resolver.
```

Comment:
I start to think that it was not a typo that you wrote `getaddr` but not sure why we would not want to mention the whole name of the method.

```suggestion
   * Specify whether to use the getaddrinfo-based system DNS resolver or the c-ares resolver.
```